databricks/koalas/groupby.py

```diff
@@ -1936,6 +1946,34 @@ def _shift(self, periods, fill_value):
                                                     for c in applied])
         return DataFrame(internal)
 
+    def describe(self):
+        kdf = self.agg(["count", "mean", "std", "min", "quartiles", "max"]).reset_index()
+        formatted_percentiles = ["25%", "50%", "75%"]
+
+        # Split "quartiles" columns into first, second, and third quartiles.
+        for label, content in kdf.iteritems():
+            if label[1] == "quartiles":
+                exploded = ks.DataFrame(
+                    {
+                        (label[0], x): _column_op(itemgetter(i))(content).to_numpy()
+                        for i, x in enumerate(formatted_percentiles)
+                    }
+                )
+                kdf = kdf.drop(label).join(exploded)
```

Comment:
`join` is expensive .. we should avoid. Actually I like @itholic's suggestion in that way.